python/ray/rllib/ppo/ppo.py

```diff
@@ -91,6 +93,7 @@ def _init(self):
         self.kl_coeff = self.config["kl_coeff"]
```

Comment:
You may want to modify L88 (`  _allow_unknown_subkeys = ["model", "tf_session_args"]`) to include `evaluator_resources`

Reply:
Yep or alternatively provide gpu:0 there